File: app/src/main/java/com/example/birdify/ui/Sightings/SightingsViewModel.java
package com.example.birdify.ui.Sightings;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class SightingsViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public SightingsViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is Sightings fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/example/birdify/ui/Sightings/SightingsViewModel.kt
package com.example.birdify.ui.Sightings

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class SightingsViewModel : ViewModel() {
    private val mText: MutableLiveData<String>

    init {
        mText = MutableLiveData()
        mText.value = "This is Sightings fragment"
    }

    val text: LiveData<String>
        get() = mText
}